citations-tool/tool/src/webapp/vm/citation/create.vm

```diff
@@ -376,14 +376,15 @@
 	 */
 	function trackPreferredUrl( checkboxId )
 	{
-		// has this been clicked?
-		if( $( "#" + checkboxId ).attr( "checked" ) != null )
+		var checkBoxClicked = $( "#" + checkboxId );
+		// has this been checked?
+		if( checkBoxClicked.attr( "checked" ) == true )
 		{
 			// clear all checkboxes
 			$( ".prefLink" ).attr( "checked", "" );
```

Comment:
You can also compact all this and just do: `$(".prefLink").not("#"+checkboxId).removeAttr("checked")` then you don't need to re-add the checked one. Doesn't need updating, just an alternative.